### PR TITLE
Adding a ped preview

### PR DESCRIPTION
**Pull request by enzo2991**
image

https://youtu.be/DcgHAqxUv9Q

### CODE REVIEW

**Review by alberttheprince — CHANGES_REQUESTED** [review]
Just requesting the removal of Citizen. and would make reviewing changes a bit easier. No longer necessary to include it unless it's for [Citizen.Await] (https://docs.fivem.net/docs/scripting-reference/runtimes/lua/functions/Citizen.Await) I believe.

**Comment by enzo2991** [comment]
yes, it's good when you want to make the thread wait for the Citizen.Wait or Await and I've just updated

**Comment by enzo2991** [comment]
They are the Same
Citizen.CreateThread is the original
CreateThread is the shorcut FiveM made for Citizen.CreateThread

**Comment by alberttheprince** [comment]
> They are the Same Citizen.CreateThread is the original CreateThread is the shorcut FiveM made for Citizen.CreateThread

Since they function the same, we should leave it as it originally was when making the PR, without Citizen. as it's become common practice to use only Wait or CreateThread without the Citizen.

We can also remove things like:

--- RPEmotes by TayMcKenzieNZ, Mathu_lmn and MadsL, maintained by TayMcKenzieNZ ---
--- Download OFFICIAL version and updates ONLY at https://github.com/TayMcKenzieNZ/rpemotes ---
--- RPEmotes is FREE and ALWAYS will be. STOP PAYING SCAMMY FUCKERS FOR SOMEONE ELSE'S WORK!!! ---


As Tay is selling copies of rpemotes now and is no longer associated with RP emotes generally

File: client/EmoteMenu.lua [hunk 1]
```diff
@@ -468,104 +547,6 @@ function AddFaceMenu(menu)
...
- 
-    infomenu:AddItem(NativeUI.CreateItem("Join the <font color=\"#00ceff\"><b>Official Discord 💬<b></font>",
-        "Join our official discord! 💬 <font color=\"#00ceff\"><b>https://discord.gg/sw3NwDq6C8<b></font>"))
-    infomenu:AddItem(NativeUI.CreateItem("Download <font color=\"#FF25B1\"><b>RPEmotes<b></font> from 💾",
-        "Official download link: <font color=\"#00ceff\">http://rpemotes.com</font>"))
-    infomenu:AddItem(NativeUI.CreateItem("Read the <font color=\"#00ceff\"><b>Official Wiki Docs 📖<b></font>",
-        "Check out our official Wiki Docs: <font color=\"#00ceff\">https://docs.rpemotes.com/</font>"))
-    infomenu:AddItem(NativeUI.CreateItem("<font color=\"#FF25B1\"><b>TayMcKenzieNZ 🇳🇿<b></font>",
-        "<font color=\"#FF25B1\">TayMcKenzieNZ 🇳🇿</font> Project Manager for RPEmotes."))
-    infomenu:AddItem(NativeUI.CreateItem("Thanks ~o~DullPear 🍐~s~", "~o~DullPear~s~ for the original dpemotes ❤️"))
```
Comment (alberttheprince) [inline client/EmoteMenu.lua]: Try to cherry pick the code you made for the animation menu.

**Comment by alberttheprince** [comment]
I've made several changes and corrections to maintain the original functionality, style, and deleted things like the comments. Would appreciate if you could give this a test run to see if it's OK.

**Comment by enzo2991** [comment]
It's ok now, I just corrected the config, the title color and outline were missing.

**Comment by TayMcKenzieNZ** [comment]
> As Tay is selling copies of rpemotes now and is no longer associated with RP emotes generally

Please stop name dropping me. I *am* still maintaining *my* project and am within my right to sell *my* work. You are the only person on earth who has a problem with it yet, you are the very reason *why* it's being sold.

**Comment by TayMcKenzieNZ** [comment]
Why are you openly name dropping me but yet delete my comments? If you have something to say, say it. My DMs are open, Noor, although I'm confused as to why you keep poking the wasp nest. It's very, very immature of you.

**Comment by TayMcKenzieNZ** [comment]
Repository owner deleted a comment from TayMcKenzieNZ 1 hour ago

**Pull request merged**